### DISCUSSION TITLE
Browser back button often does not work between PR tabs

### DISCUSSION THREAD

**Discussion by sarayourfriend** [community/community#29737] [Pull Requests]
I've asked my coworkers to corroborate this experience and several of us have the same issue. When navigating on a PR, very often when hitting the browser back button to go from the files tab back to the PR discussion tab or vice-versa does nothing. It seems like the page just crashes. For example, very often when I hit the back button from the files tab to try to go back and read the discussion, nothing will happen (for a while) so I'll hit the back button again and I'll end up back on the notifications page.

I have a hard time intentionally reproducing this, but it happens so often during the day that I am desperately looking for alternative ways to review PRs that do not involve GitHub's web UI.

**Comment by airtower-luna** [comment]
> I have a hard time intentionally reproducing this, but it happens so often during the day that I am desperately looking for alternative ways to review PRs that do not involve GitHub's web UI.

For what it's worth, you can `git fetch` the PR branch, and then check `git diff` locally. But of course that doesn't let you add per-line notes on the PR, so I don't know if that's sufficient for you.

Personally I often work with two browser tabs for reviews, one for the diff and one for the comments.

**Reply by sarayourfriend** [reply]
I'm aware of how to review the changes proposed in a PR without the git UI but I cannot leave comments or suggested changes without the git UI.

The browser tabs is also my work around but if you're like me and get lost when too many tabs are open, it's not a very pleasant one.

**Comment by rafalkrupinski** [comment]
Also going back from PR to other tabs in the project (issues, code, etc) doesn't work.

**Reply by rafalkrupinski** [reply]
Can someone add Bug label pls?
